cylc/flow/host_select.py

```diff
@@ -533,7 +533,7 @@ def _get_metrics(hosts, metrics, data=None):
     }
     for host in hosts:
         if is_remote_host(host):
-            proc_map[host] = _remote_cylc_cmd(cmd, host=host, **kwargs)
+            proc_map[host] = cylc_server_cmd(cmd, host=host, **kwargs)
```

Comment:
Behaviour change: this now uses the localhost platform config.

It now also logs the command to debug level.